packages/wrangler/src/containers/registries.ts

```diff
@@ -106,83 +119,107 @@ async function registryConfigureCommand(
 
 	log(`Configuring ${registryType.name} registry: ${configureArgs.DOMAIN}\n`);
 
-	if (configureArgs.secretName) {
-		validateSecretName(configureArgs.secretName);
-	}
-	let secret: string;
-	switch (registryType.type) {
-		case "cloudflare":
-			log(
-				"You do not need to configure credentials for Cloudflare managed registries.\n"
-			);
-			endSection("No configuration required");
-			return;
-		// this can be extended to any registry type that requires credentials
-		case ExternalRegistryKind.ECR:
-			log(`Getting ${registryType.secretType}...\n`);
-			secret = await getSecret(registryType.secretType);
-			break;
-		default:
-			throw new UserError(`Unhandled registry type: ${registryType.type}`);
+	if (registryType.type === "cloudflare") {
+		log(
+			"You do not need to configure credentials for Cloudflare managed registries.\n"
+		);
+		endSection("No configuration required");
+		return;
 	}
 
-	log("\n");
-	log("Setting up integration with Secrets Store...\n");
-	const accountId = await getAccountId(config);
-	let secretStoreId = configureArgs.secretStoreId;
-	if (!secretStoreId) {
-		const stores = await listStores(config, accountId);
-		if (stores.length === 0) {
-			const defaultStoreName = "default_secret_store";
-			const check = await confirm(
-				`No existing Secret Stores found. Create a Secret Store to store your registry credentials?`
-			);
-			if (!check) {
-				endSection("Cancelled.");
-				return;
-			}
-			const res = await promiseSpinner(
-				createStore(config, accountId, { name: defaultStoreName })
-			);
-			log(`New Secret Store ${defaultStoreName} created with id: ${res.id}`);
-			secretStoreId = res.id;
-		} else if (stores.length > 1) {
-			// note you can only have one secret store per account for now
+	const isFedRAMPHigh =
+		getCloudflareComplianceRegion(config) === "fedramp_high";
+	if (isFedRAMPHigh) {
+		if (!configureArgs.disableSecretsStore) {
 			throw new UserError(
-				`Multiple Secret Stores found. Please specify a Secret Store ID using --secret-store-id.`
+				"Secrets Store is not supported in FedRAMP compliance regions. You must set --disableSecretsStore."
```

Comment:
Out of curiosity: why don't we disable secret store automatically in FedRAMP compliance regions? Are we trying to make this more explicit to users?